File: daq/slc/extra/javalib/b2daq/src/b2daq/dqm/core/Graph1I.java
package b2daq.dqm.core;


public class Graph1I extends Graph1 {

	public Graph1I() {
		super();
		data = new IntArray(0);
	}
	public Graph1I(String name, String title, int nbinx, double xmin, double xmax,
			double ymin, double ymax) {
		super(name, title, nbinx, xmin, xmax, ymin, ymax);
		data = new IntArray(nbinx*2);
	}
	public Graph1I(Graph1I h) {
		super();
		data = new IntArray(0);
	}
	public String getDataType() {
		return "g1I";
	}

}

File: daq/slc/dqm/src/Graph1I.cc
#include "daq/slc/dqm/Graph1I.h"

#include "daq/slc/dqm/IntArray.h"

using namespace Belle2;

Graph1I::Graph1I() : Graph1()
{
  m_data = new IntArray(0);
}

Graph1I::Graph1I(const Graph1I& h) : Graph1(h)
{
  m_data = new IntArray(*h.m_data);
}

Graph1I::Graph1I(const std::string& name, const std::string& title,
                 int nbinx, double xmin, double xmax, double ymin, double ymax)
  : Graph1(name, title, nbinx, xmin, xmax, ymin, ymax)
{
  m_data = new IntArray(nbinx * 2);
}

Graph1I::~Graph1I() throw() {}

std::string Graph1I::getDataType() const throw()
{
  return std::string("g1I");
}

